specification/electronic-prescription-service-api.yaml

```diff
@@ -177,62 +178,23 @@ info:
-    | Any                   | 400       | See "Unsupported operation" example below | Sandbox          | Unsupported operation |
-    | Any                   | 408       | Returns original request data             | Integration test | Request timeout       |
-
-
-    #### Example: Unsupported operation
-    If attempting to call the `/_poll` endpoint in the sandbox environment, the following error would be returned:
-      ```
-      {
-        resourceType: "OperationOutcome",
-        issue: [
-          {
-            code: "informational",
-            severity: "information",
-            details: {
-              coding: [
-                {
-                  code: "INTERACTION_NOT_SUPPORTED_BY_SANDBOX",
-                  display: "Interaction not supported by sandbox",
-                  system: "https://fhir.nhs.uk/R4/CodeSystem/Spine-ErrorOrWarningCode",
-                  version: "1"
-                }
-              ]
-            }
-          }
-        ]
-      }
-      ```
-
-    #### Validation errors
-    Our endpoints can return the following errors:
-
-    | HTTP code | Issue severity | Issue type | Error code             | Cause                                                           |
-    | ----------| -------------- | ---------- | ---------------------- | --------------------------------------------------------------- |
-    | 400       | error          | value      | MISSING_FIELD          | Bundle contains incorrect number of resources of a certain type |
-    | 400       | error          | value      | MISSING_FIELD          | Bundle does not contain "id" field                              |
-    | 400       | error          | value      | INVALID_VALUE          | Invalid value in MedicationRequest                              |
-    | 400       | fatal          | value      | MISSING_FIELD          | Bundle does not contain "entry" field                           |
-    | 400       | fatal          | value      | INCORRECT_RESOURCETYPE | ResourceType is not "Bundle"                                    |
-    | 400       | fatal          | value      | INVALID_VALUE          | MessageHeader indicates invalid MessageType                     |
-    | 400       | fatal          | value      | MISSING_FIELD          | X-Request-ID header missing or malformed                     |
-
 
+    ## Errors
+    ### API specific errors
+    | HTTP Status   | Error Code             | Description                                    |
+    | --------------| ---------------------- | ---------------------------------------------- |
+    | 400           | MISSING_FIELD          | A required field is missing within the request |
+    | 400           | INVALID_VALUE          | Invalid value in message                       |
+    | 400           | INCORRECT_RESOURCETYPE | FHIR resource has incorrect resourceType       |
+    | 404           | NOT_FOUND              | Resource not found                             |
```

Comment:
I think `404` can be removed from the API-specific section

Reply:
this is here since a different error is returned from Apigee vs Hapi 404s, not much value here tho so can remove :+1: